src/components/app-footer/app-footer.vue

```diff
@@ -43,44 +43,60 @@
           </li>
         </ul>
       </nav>
-      <nav
-        ref="contact"
-        class="app-footer__column app-footer__column--right"
-        aria-label="Contact"
-      >
-        <ul class="body-detail app-footer__list app-footer__list--contact">
-          <li
-            v-for="address in app.addresses"
-            :key="address.address"
-            class="app-footer__list-item app-footer__list-item--address"
-          >
-            <a
-              :href="address.googleMapsLink"
-              class="app-footer__link app-footer__link--right"
-              target="_blank"
-              rel="noreferrer noopener"
+
+      <div>
+        <h2 class="h4 app-footer__title">
+          {{ $t('say_hello') }}
+        </h2>
+        <nav
+          aria-label="Contact"
+        >
```

Comment:
You can wrap the `h2` inside the `nav` element and then remove the `aria-label` from the nav element. Now it's kind of double labeled.